integration/internal/network/network.go

```diff
@@ -0,0 +1,39 @@
+package network
+
+import (
+	"context"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/gotestyourself/gotestyourself/assert"
+)
+
+// TestNetworkConfig holds network configuration struct
+// used in api calls
+type TestNetworkConfig struct {
```

Comment:
This is struct should not be required. Looking at `Create`, `Name` is required and thus, should be an argument on it's own.

Reply:
Thank you for the review. I will update the PR.